### DISCUSSION TITLE
Soroban + Rust developer guides for JavaScript developers

### DISCUSSION THREAD

**Discussion by wildework** [stellar/sorobanathon#60] [General]
I'm a software developer, with my main stack being JavaScript, and as soon as I started reading the Soroban docs it hit me - I'm going to struggle trying to learn Soroban without a gaining a good grasp of Rust. I like to learn by digging deeper and so I started documenting the things I learn while going through the first few tutorials.

The result is a Soroban developer portal designed to onboard web developers, at least I hope to design it to be so.

[www.sorobandev.com](https://sorobandev.com/)

I'm submitting the first two tutorials with Soroban + Rust explainers and I plan on doing many more. Maybe even with video content alongside.

I hope these are useful.

**Comment by rawritude** [comment]
Not rust oriented so having resources that break down the language components in soroban is much appreciated. Thank you!

**Comment by achallagundla** [comment]
Woah This is neat!! Thank you for your submission, will get back to you shortly once we have fully reviewed your submission!

**Comment by achallagundla** [comment]
Heya @wildework  ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1000 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
